doc/api/n-api.md

```diff
@@ -2492,6 +2492,35 @@ This API creates a JavaScript `symbol` value from a UTF8-encoded C string.
 The JavaScript `symbol` type is described in [Section 19.4][]
 of the ECMAScript Language Specification.
 
+#### `node_api_symbol_for`
+
+<!-- YAML
+added: REPLACEME
+napiVersion: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+napi_status node_api_symbol_for(napi_env env,
+                                const char* description,
+                                napi_value* result)
+```
+
+* `[in] env`: The environment that the API is invoked under.
+* `[in] description`: C string representing the text to be used as the
```

Comment:
```suggestion
* `[in] description`: UTF-8 C string representing the text to be used as the
```

(a number of APIs also name their argument in a way that reflects this, e.g. `utf8Name` in `napi_set_named_property()`/`napi_get_named_property()`)

Reply:
Thx, updated!